guide/designing-products/lightning-service-providers.md

```diff
@@ -0,0 +1,306 @@
+---
+layout: guide
+title: Lightning Service Providers
+description: Common user experience problems of the Lightning network that can be resolved by third-party services.
+nav_order: 10
+parent: Designing bitcoin products
+permalink: /guide/designing-products/lightning-service-providers/
+main_classes: -no-top-padding
+---
+
+<!--
+
+Editor's notes
+
+Illustration sources: https://www.figma.com/file/qr4P17z6WSPADm6oW0cKw2/?node-id=701%3A5911
+
+-->
+
+{% include picture.html
+   image = "/assets/images/guide/designing-products/lightning-service-providers/lightning-service-providers.jpg"
+   retina = "/assets/images/guide/designing-products/lightning-service-providers/lightning-service-providers@2x.jpg"
+   mobile = "/assets/images/guide/designing-products/lightning-service-providers/lightning-service-providers-mobile.jpg"
+   mobileRetina = "/assets/images/guide/designing-products/lightning-service-providers/lightning-service-providers@2x.jpg"
+   alt-text = "Lightning service providers header image."
+   width = 1600
+   height = 740
+   layout = "full-width"
+%}
+
+# Lightning Service Providers
+
+The lightning network is our best solution to bitcoins lack of scalability. However, using the lightning network self-custodially brings about many new usability issues, such as complex channel management and backups.
+
+Lightning service providers (LSPs) aim to solve these issues. They act as non-custodial third parties that bitcoin lightning wallets can connect to and use services to automate various lightning operations.
+
+The goal of LSPs, [as put by Roy Sheinfeld](https://medium.com/breez-technology/envisioning-lsps-in-the-lightning-economy-832b45871992) of Breez, is to assist with onboarding the world to the Lightning economy.
+
+## Lightning Services
+
+Lightning services are offered by LSPs to solve a particular pain point when using a non-custodial bitcoin lightning wallet.
```

Comment:
> I think the point is that when the wallet is custodial, you don't need an LSP

Yes this is correct. I can see how this could be confusing, will re-word it so it's more clear. LSPs are non-custodial yes, they are only used as 'hand-holders' for custodial wallets but never take custody. Some LSPs do however, say when doing a trusted swap. But they could (should) be using submarine swaps which are non-custodial. Attempted to stress that point in the swap sections of this page